app/components/Map.js

```diff
@@ -131,32 +149,32 @@ export default class Map extends Component {
     )
   }
 
-
-  moveMapToUser(location) {
-    const {currLoc} = this.props;
-    let currRegion = {
-      latitude: currLoc.latitude,
-      longitude: currLoc.longitude,
-      latitudeDelta: 0.005,
-      longitudeDelta: 0.005
-    };
-    this.refs.map.animateToRegion(currRegion, 250)
+  moveMapToUser() {
+    const {stateLocation} = this.state;
```

Comment:
Since `stateLocation` is set only on `componentDidMount`, the "center on me" button won't work if the user moves while the Map is open. They'd have to move quite a bit to notice at the default zoom level, so maybe this is a non-issue, but it's worth noting.